### PR TITLE
add Socrata organization, add Socrata source

### PR DESCRIPTION
**Pull request by PeterAustinMoore**
- Adding Socrata Organization
- Adding Socrata Source

### CODE REVIEW

**Comment by PeterAustinMoore** [comment]
I signed it.

File: sources/socrata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "SOCRATA",
+  "name": "Socrata",
+  "categories": ["ANALYTICS", "PUBLIC_DATA"],
+  "organization": "SOCRATA",
+  "iconUrl":
+    "https://opendata.socrata.com/api/assets/BFCF469C-E3F8-44A1-9E5A-032AF3457AFB?LogoSwirl.png",
+  "sourceUrl": "https://github.com/socrata/data-studio-connector",
```
Comment (pfrisella) [inline sources/socrata.json]: sourceUrl should point to where this data comes from, this is unrelated to the connector (e.g. in general multiple connectors could use the same source). If a user wanted to learn more about this source which is the best URL to send them? I suspect it may just be https://www.socrata.com/ in this case.

File: sources/socrata.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "dataVisibility": ["PRIVATE"]
```
Comment (pfrisella) [inline sources/socrata.json]: Can you confirm there is no Public data.

Review the description for dataVisibility in the following table:
https://github.com/googledatastudio/ds-data-registry#add-a-source

This is not about auth or access to the data but what type of data it is. Consider 2 separate users using this source, would they only be able to query or data specific/private to them or is it possible that they could both query for the same data. The latter would be considered public data.

**Comment by pfrisella** [comment]
Can you also review and resolve the CLA issue.
If you want to check which email is associated with the commits you can view the patch file by adding .patch to the pull request URL: https://github.com/googledatastudio/ds-data-registry/pull/25.patch

File: sources/socrata.json (hunk 2 above)
Comment (PeterAustinMoore) [reply sources/socrata.json]: Ok - I will change that to both public and private

**Comment by PeterAustinMoore** [comment]
@pfrisella those issues have been resolved
